### ISSUE TITLE
closestKodein is not working with androidx.fragment.app.Fragment

### ISSUE THREAD

**Issue by Amirgg** [Kodein-Framework/Kodein-DI#218]
im using 

```
implementation 'org.kodein.di:kodein-di-generic-jvm:6.3.0'
implementation 'org.kodein.di:kodein-di-framework-android-x:6.3.0'
```

and here's the fragment class
```
...
import androidx.fragment.app.Fragment
import org.kodein.di.KodeinAware
import org.kodein.di.android.closestKodein

class LoginFragment : Fragment(), KodeinAware {
    override val kodein by closestKodein()
...
}
```

is says:
Property delegate must have  a 'ProvideDelegate...

what am i missing?

**Comment by romainbsl** [comment]
Sorry, this is a bug. I will correct this asap

**Comment by romainbsl** [comment]
`closestKodein`  is now available for `android-x` in `6.3.1`

**Comment by Rasalexman** [comment]
you don't need to use closestKodein in android anymore. Just use `by kodein(appContext)`

**Comment by rodrigmatrix** [comment]
I'm having the same issue with kodein 6.3.3
I'm using the androidx module:

```
import androidx.fragment.app.Fragment

class RequestLoanFragment : Fragment(), KodeinAware {
    override val kodein by closestKodein()
}
```
and it still asks for a context

> `closestKodein` is now available for `android-x` in `6.3.1`

**Comment by romainbsl** [comment]
What kind of import do you have for `closestKodein` in your class ?
